test/e2e/infra/vagrant/playbook/roles/common/tasks/docker.yml

```diff
@@ -33,15 +33,15 @@
     groups: docker
     append: yes
     create_home: yes
-  when: ansible_version.full >= "2.5.0"
+  when: ansible_version.full is version_compare('2.5.0', '>=')
 
 - name: Add test user to docker group (Ansible < 2.5.0)
   user:
     name: "{{ test_user }}"
     groups: docker
     append: yes
     createhome: yes
-  when: ansible_version.full < "2.5.0"
+  when: ansible_version.full is version_compare('2.5.0', '<')
```

Comment:
Could you please explain why we need this change?
If there is an issue with these two lines, I would suggest having a separate PR to fix.